test/Microsoft.Windows.ComputeVirtualization.Tests/project.json

```diff
@@ -10,11 +10,11 @@
     "debugType": "portable"
   },
   "runtimes": {
-    "win10-x64": {}
+    "win81-x64": {}
```

Comment:
I don't think we have a good way.  JSON does not support comments, so we can't add a reminder.